dataset/default/tool.json.example

```diff
@@ -0,0 +1,40 @@
+{
+  "slug": "app-name",
+  "name": "app",
+  "tagline": "Lorem ipsum dolor sit amet.",
+  "description": "Lorem ipsum dolor sit amet, consectetur adipiscing elit. Nullam tincidunt leo sed nunc tincidunt auctor. Orci varius natoque penatibus et magnis dis parturient montes, nascetur ridiculus mus. Praesent rutrum eget ex eget tempor.",
+  "website_url": "https://app.com",
+  "logo": "https://i.imgur.com/nVHYQWU.png",
+  "cover_img": "app.png",
+  // development, testnet, mainnet
+  "network": "testnet",
+  "developer": {
+    "name": "GOOD_DEV",
```

Comment:
This placeholder formatting is potentially confusing. "developer name" might be clearer to follow the pattern above.